.storybook/postcss.config.js

```diff
@@ -1,25 +1,22 @@
 /* eslint-disable global-require, @typescript-eslint/no-var-requires */
-
+const path = require('path')
 module.exports = {
     parser: 'postcss-comment',
-    plugins: () => {
-        const res = [
+    plugins:  [
             require('postcss-easy-import')({
                 path: ['../src', 'node_modules'],
             }),
-            require('tailwindcss'),
+            require('tailwindcss')({ config: path.resolve(__dirname, 'tailwind.config.js') }),
```

Comment:
Is it ok using `.js` here?

Reply:
This is storybook config, so it's no problem :)